src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -1528,3 +1528,14 @@ def get_multisite_status(self):
                 and len(rgw_zone_list['zones']) < 1:
             is_multisite_configured = False
         return is_multisite_configured
+
+    def get_multisite_sync_status(self):
+        rgw_multisite_sync_status_cmd = ['sync', 'status']
+        try:
+            exit_code, out, _ = mgr.send_rgwadmin_command(rgw_multisite_sync_status_cmd, False)
+            if exit_code > 0:
+                raise DashboardException('Unable to get multisite sync status',
+                                         http_status_code=500, component='rgw')
+            return out
+        except SubprocessError as error:
+            raise DashboardException(error, http_status_code=500, component='rgw')
```

Comment:
Here is where all the processing/parsing/reformatting should happen, and not in the front-end :)

Reply:
Moved code to backend